pyronear/datasets/openfire.py

```diff
@@ -1,5 +1,9 @@
 # -*- coding: utf-8 -*-
 
+# Copyright (c) The pyronear developers.
+# This file is dual licensed under the terms of the CeCILL-2.1 and GPLv3 licenses.
+# See the LICENSE file in the root of this repository for complete details.
+
```

Comment:
The arguments in the [GNU FAQ](https://www.gnu.org/licenses/gpl-faq.en.html#LicenseCopyOnly) you mentioned are very clear

>Just putting a copy of the GNU GPL in a file in your repository does not explicitly state that the code in the same repository may be used under the GNU GPL. Without such a statement, it's not entirely clear that the permissions in the license really apply to any particular source file. **An explicit statement saying that eliminates all doubt.**

>This statement should be in each source file. **A clear statement in the program's README file is legally sufficient as long as that accompanies the code, but it is easy for them to get separated**. Why take a risk of uncertainty about your code's license?

> You should put a notice at the start of each source file, stating what license it carries, **in order to avoid risk of the code's getting disconnected from its license**. If your repository's README says that source file is under the GNU GPL, what happens if someone copies that file to another program? That other context may not show what the file's license is. It may appear to have some other license, or no license at all (which would make the code nonfree).

> Adding a copyright notice and a license notice **at the start of each source file is easy and makes such confusion unlikely**.